src/features/embalm/stepContent/steps/TotalRequiredArchaeologists.tsx

```diff
@@ -72,40 +44,31 @@ export function TotalRequiredArchaegologists() {
         mt={6}
         size="sm"
       >
-        Assign Total Archs and Required Archs
+        Assign Number of Required Archaeologists
       </Heading>
+      <Text
+        mt={6}
+        size="xs"
+      >
+        Total Archaeologists Selected: {totalArchaeologists}
+        {!totalArchaeologists && (
+          <Text
+            mt={3}
+            color="error"
+          >
+            Please select at least one archaeologist from the Select Archaeologists page
+          </Text>
+        )}
+      </Text>
       <Divider my={6} />
       <Text>
-        How many archaeologists would you like to assign to this Sarcophagus and how many are
-        required to unwrap?
+        How many archaeologists are
+        required to unwrap the sarcophagus?
       </Text>
       <FormControl>
         <Flex mt={6}>
-          <Flex
-            direction="column"
-            w="200px"
-          >
-            <FormLabel color={inputColor}>Total Archaeologists</FormLabel>
-            <NumberInput
-              allowMouseWheel
-              w="100px"
-              color={inputColor}
-              value={totalArchaeologists}
-              onChange={handleTotalArchaeologistsChange}
-            >
-              <NumberInputField
-                borderColor={inputColor}
-                color="brand.950"
-              />
-              <NumberInputStepper>
-                <NumberIncrementStepper borderColor={inputColor} />
-                <NumberDecrementStepper borderColor={inputColor} />
-              </NumberInputStepper>
-            </NumberInput>
-          </Flex>
           <Flex direction="column">
-            <FormLabel color={inputColor}>Archaeologists required to unwrap</FormLabel>
-            <NumberInput
+            <NumberInput defaultValue={1} min={1} max={totalArchaeologists || 1}
```

Comment:
This input could use some more polish. When you click on the input and backspace it displays "NaN".

Reply:
This is probably not even related to this PR.

Reply:
@knoll3 -- As soon as it "un-blurs", it will reset the value from `NaN` to the `defaultValue`, which is good, but this should get looked into further.